detection models without ir problems

ysyuann opened the pull request:
PicoDet-XS  
PicoDet-M 
FCOS-ResNet50
DETR-R50
PP-YOLOE-vehicle-L
PP-YOLOE-vehicle-S
PicoDet-LCNet_x2_5
PP-YOLOE-human-S
PP-YOLOE-human-L
PP-YOLOE+SOD系列

TingquanGao committed:
update models name (ch_SVTRv2_rec, ch_RepSVTR_rec)

ysyuann committed:
detection models without ir problems

cuicheng01 commented on paddlex/repo_apis/PaddleDetection_api/object_det/register.py:
```diff
@@ -350,3 +350,147 @@
+
+register_model_info(
+    {
+        'model_name': 'PicoDet-LCNet_x2_5',
```
这个名字建议改为`PP-ShiTuV2_det`

cuicheng01 commented on paddlex/repo_apis/PaddleDetection_api/object_det/register.py:
```diff
@@ -350,3 +350,147 @@
+    {
+        'model_name': 'PicoDet-LCNet_x2_5',
+        'suite': 'Det',
+        'config_path': osp.join(PDX_CONFIG_DIR, 'PicoDet-LCNet_x2_5.yaml'),
```
yaml同上

cuicheng01 commented on paddlex/repo_apis/PaddleDetection_api/object_det/register.py:
```diff
@@ -350,3 +350,147 @@
+
+register_model_info(
+    {
+        'model_name': 'PP-YOLOE-human-L',
```
这块建议改为`PP-YOLOE-L_human`，vehicle同理

ysyuann committed:
Merge branch 'develop' of github.com:PaddlePaddle/PaddleX into for_detection

ysyuann committed:
export model path

ysyuann committed:
change names

ysyuann committed:
change names

ysyuann committed:
change predict imgs

ysyuann committed:
add CenterNet models

Sunting78 committed:
Merge branch 'develop' into for_detection

Sunting78 committed:
add uvdoc (#1991)

* add uvdoc

* add uvdoc

TingquanGao committed:
update pp option

1. support option.enable_new_ir
2. rename
3. update args

TingquanGao committed:
upgrade PaddleX Inference

1. update API create_predictor;
2. support ML Clas;
3. support pass pp_option;
4. unify key to 'result';

changdazhou committed:
support table recognition predictor

Bobholamovic committed:
[Refactor] New abstraction for predictors (#1990)

* Refactor base predictor

* Fix bug

* Add config_path attr

* Change create_predictor signature

* Fix class

TingquanGao committed:
update paddlex inference

1. fix json numpy.array
2. fix when no text detection in ocr
3. fix key to result

ysyuann committed:
add det model

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/PaddlePaddle/PaddleX?pullRequest=1982) <br/>All committers have signed the CLA.
